### DISCUSSION TITLE
Got error in the new milvus version: module 'pymilvus.orm.utility' has no attribute 'calc_distance' stackTrace

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#22943] [Q&A and General discussion]
Got error in the new milvus version: module 'pymilvus.orm.utility' has no attribute 'calc_distance' stackTrace

How to properly compare two vectors without loading them.

**Comment by xiaofan-luan** [comment]
we no longer support calc_distance

**Reply by martinenkoEduard** [reply]
So it is not possible to compare two vectors?

**Reply by martinenkoEduard** [reply]
How do I emulate it then? 
So the answer is equal the milvus's answer (because I use logic to calculate percentage with regards of distance).

You can write a simple method to calculate.
For example, to calculate L2 distance between vectors
```
import numpy as np
def distance(v1, v2):
    return np.sqrt(np.sum((v1 - v2) ** 2))   
```

**Reply by martinenkoEduard** [reply]
Does mivlus also applied np.sqrt function? So I get the same exact distance?
